project_precommit_check

```diff
@@ -100,6 +100,13 @@ supported_configs = {
                        'Target: x86_64-apple-darwin18.2.0\n',
             'description': 'Xcode 11.5 (contains Swift 5.2.4)',
             'branch': 'swift-5.2-branch'
+        },
+        '5.3': {
+            'version': 'Apple Swift version 5.3.2 '
+                       '(swiftlang-1200.0.45 clang-1200.0.32.28)\n'
+                       'Target: x86_64-apple-darwin18.2.0\n',
+            'description': 'Xcode 12.5 (contains Swift 5.3.3)',
```

Comment:
Should the description be 5.3.2 not 5.3.3?

Reply:
Oops, right, Xcode 12.5 is in beta. The target spec also doesn't match an OS that Xcode 12.4 deploys to.